src/fdk_rdf_parser/classes/conforms_to.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+from typing import Dict, Optional
+
+
+@dataclass
+class ConformsTo:
+    uri: Optional[str] = None
+    prefLabel: Optional[Dict[str, str]] = None
```

Comment:
Eg synes det hadde vore meir ryddig å kalle denne variabelen "title" i staden for "prefLabel".

Reply:
Ja, det er det. Men forstod det på deg i går at det var bedre å bare fortsette med samme json-struktur vi har hatt så langt, kan dog endre til title og ta hånd om feilene som da oppstår i portal. Skal jeg heller gjøre det?